### ISSUE TITLE
As a SERGE maintainer I wish to get user feedback about the gameplay experience

### ISSUE THREAD

**Issue by IanMayo** [serge-web/serge#1320]
To address overall usability and potential bottlenecks we've agreed to conduct a survey of Serge players/stakeholders.

The working copy is [here](https://docs.google.com/forms/d/1OLBwxySzfrHh5RRK44dvLyk6STgJgG_1lK0Uyf2hJeA/edit?ts=61e31788) though note it may not be publicly visible in that form.

This issue contains feedback that I was unable to give within the questionnaire itself.

We give the scoring system twice, very close to each other. I guess we can _lose_ the first one.
![image](https://user-images.githubusercontent.com/1108513/150811341-ad010cc7-c985-45e3-92b0-4799ba3f4e6f.png)

**Comment by ShendMikullovci** [comment]
Removed the 1st scoring table. I also approve of both of the short descriptions for the question types.

**Comment by ShendMikullovci** [comment]
Any news on this?

**Comment by IanMayo** [comment]
> Any news on this?

Just double-checking that client is happy for us to send this out to participants of most recent game.

**Comment by ShendMikullovci** [comment]
> > Any news on this?
> 
> Just double-checking that client is happy for us to send this out to participants of most recent game.

Have they replied? Curious to see the answers.

**Comment by IanMayo** [comment]
> Have they replied? Curious to see the answers.
No.  I've been given a "cleaned-up" contact list to use.  Don't think I'll get chance to do it tomorrow, will try to on Monday.

When it does happen, I'm sure you'll start getting emails for "form completed", or something like that.
